components/mediation/mediators/dataservices-mediator/org.wso2.micro.integrator.mediator.dataservice/src/main/java/org/wso2/micro/integrator/mediator/dataservice/DataServiceCallMediatorConstants.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  you may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+package org.wso2.micro.integrator.mediator.dataservice;
+
+public class DataServiceCallMediatorConstants {
+
+    public static final String PAYLOAD_NAME_SPACE_URI = "http://ws.wso2.org/dataservice";
+    public static final String PAYLOAD_PREFIX = "axis2ns";
+    public static final String TARGET_PROPERTY_TYPE = "property";
+    public final static String JSON_TYPE = "json";
+    public final static String XML_TYPE = "xml";
+    public static final String DATA_SERVICES_CALL = "dataServicesCall";
```

Comment:
since this can call only one data service , shouldn't this be dataServiceCall

Reply:
Since we refer dataservices generally, it is named as that